### DISCUSSION TITLE
Website is not displayed, only [ ]

### DISCUSSION THREAD

**Discussion by Jorge-Nunes** [mediagis/nominatim-docker#256] [General]
When the installation is finished, the website is not displayed, only [ ].

![image](https://user-images.githubusercontent.com/48777260/132240295-7691bd6f-f880-458b-a657-2d100d7db08f.png)

**Comment by mtmail** [comment]
What website are you expecting? Nominatim only returns data (JSON, XML). https://github.com/osm-search/nominatim-ui is a separate project, not part of this docker project.

**Comment by leonardehrenfried** [comment]
Seeing that it's just a bunch of HTML/JS, if someone is interested in adding this to the image, I'd be happy to merge it.

**Reply by Stefanic** [reply]
I've looked into this, biggest issue is setting the correct API endpoint. You'll need to set the external API IP/URL in the config inside the container.
